npm/private/npm_package.bzl

```diff
@@ -65,64 +72,57 @@ If unset, a npm_link_package that references this npm_package must define the pa
+        Files in srcs are only copied to the output directory if
+        the Bazel package of the file matches one of the patterns specified.
 
         Forward slashes (`/`) should be used as path separators.
 
         A "." value expands to the target's package path (`ctx.label.package`).
         A "./**" value expands to the target's package path followed by a slash and a
         globstar (`"{{}}/**".format(ctx.label.package)`).
 
-        Defaults to [".", "./**"] which includes sources target's package and subpackages.
+        Defaults to ["./**"] which includes sources target's package and subpackages.
 
         Files and directories that have matching Bazel packages are subject to subsequent filters and
         transformations to determine if they are copied and what their path in the output
         directory will be.
 
+        {glob_support_docstring}
+
         See `copy_to_directory_action` documentation for list of order of filters and transformations:
-        https://github.com/aspect-build/bazel-lib/blob/main/docs/copy_to_directory.md#copy_to_directory.
-        """,
+        https://docs.aspect.build/rules/aspect_bazel_lib/docs/copy_to_directory.
+        """.format(
+            glob_support_docstring = _glob_support_docstring,
+        ),
     ),
     "exclude_srcs_patterns": attr.string_list(
         default = [
-            "node_modules/**",
             "**/node_modules/**",
         ],
         doc = """List of paths (with glob support) to exclude from output directory.
 
-        Glob patterns `**`, `*` and `?` are supported. See `glob_match` documentation for
-        more details on how to use glob patterns:
-        https://github.com/aspect-build/bazel-lib/blob/main/docs/glob_match.md.
-
         Files and directories in srcs are not copied to the output directory if their output
         directory path, after applying `root_paths`, matches one of the patterns specified.
 
-        Patterns do not look into files within source directory or generated directory (TreeArtifact)
-        targets since matches are performed in Starlark. To use `include_srcs_patterns` on files
-        within directories you can use the `make_directory_paths` helper to specify individual files inside
-        directories in `srcs`. This restriction may be fixed in a future release by performing matching
-        inside the copy action instead of in Starlark.
-
         Forward slashes (`/`) should be used as path separators.
 
-        Defaults to ["node_modules/**", "**/node_modules/**"] which excludes all node_modules folders
+        Defaults to ["**/node_modules/**"] which excludes all node_modules folders
         from the output directory.
 
         Files and directories that do not have matching output directory paths are subject to subsequent
         filters and transformations to determine if they are copied and what their path in the output
         directory will be.
 
+        {glob_support_docstring}
```

Comment:
nit: it's a lot of text for attributes, maybe this goes in separate markdown?

Reply:
yeah; it should get cleaned up soon starting with upstream. The rendering of these is completely broken on the docsite right now